dockerfiles/openedx-edxapp/Dockerfile

```diff
@@ -0,0 +1,139 @@
+# Heavily inspired by Tutor from overhang.io
+
+# This file and other files adapted from Tutor are covered by and subject to the AGPL-3
+# as a clause 5 'aggregate'.
+
+# This dockerfile is not expected to work locally. Only executes correctly within the context
```

Comment:
We could do a flag as suggested here https://github.com/concourse/concourse/issues/2105#issuecomment-472135872 but I think it would add a lot of complexity to the dockerfile? 🤷

Reply:
Probably not worth it. I was hoping for low hanging fruit like a `$CONCOURSE` env variable we could check. I wasn't aware the concourse folks had such Strong Opinions :)